rollup.config.js

```diff
@@ -2,21 +2,68 @@ import babel from 'rollup-plugin-babel'
 import css from 'rollup-plugin-css-porter'
 import pkg from './package.json'
 
+const fs = require('fs')
+
+/**
+ * Custom rollup plug-in which bundles all transforms' CSS in a single file.
+ * @param  {callback} options Rollup plug-in options func
+ * @return {void}
+ */
+const wikimediaPageLibraryTransformCSSRollupPlugin = (options = {}) => {
```

Comment:
> Importing (or requiring) ThemeTransform.css in index.js before all other imports. I actually wouldn't mind moving all CSS imports to index.js since order matters.

I like this much better... switching and I'll remove the custom rollup plugin.